target/scripts/startup/fixes-stack.sh

```diff
@@ -35,21 +33,18 @@ function _fix_var_amavis_permissions
   [[ ${ONE_DIR} -eq 0 ]] && AMAVIS_STATE_DIR="/var/lib/amavis"
   [[ ! -e ${AMAVIS_STATE_DIR} ]] && return 0
 
-  _notify 'inf' 'Checking and fixing Amavis permissions'
-  chown -hR amavis:amavis "${AMAVIS_STATE_DIR}"
-
-  return 0
```

Comment:
I'm not sure what the `return 0` was for here, I see you removed it elsewhere too, but not earlier in this function which calls `return 0` if conditional was true. Was that missed for removal, or is that particular case still valid for `return 0`?

Reply:
See my other comment. Return 0 was pretty pointless. In that case for example, `chown` could fail, but return 0 ensures, that the function never the less "reports back", everything is ok.

Reply:
The mistake in thinking here is, that the function stops on the first error. That is not the case. Even when a command fails, all following commands are executed.

Reply:
To give my two cents here as well, especially when using `set -euEo pipefail` (mentioned in another comment today or yesterday), you may need a `return 0` because Bash will return from a function with the return value of the last command of not explicit `return 0` is provided. Therefore, when running Bash in "strict mode", you may actually _need_ a `return 0` in some places.